extensions/community/PointAndOrbit.json

```diff
@@ -1,15 +1,29 @@
 {
   "author": "",
   "category": "Movement",
-  "description": "# User can\n- Use the extension action multi times.\n- Point object toward position.\n- Orbit object around position.\n- Set the distance to orbit around.\n- Set the pointing speed.\n\n# Example\n![](https://i.imgur.com/N2KtaMT.png)",
   "extensionNamespace": "",
   "fullName": "Point and Orbit",
   "helpPath": "",
   "iconUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48IURPQ1RZUEUgc3ZnIFBVQkxJQyAiLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4iICJodHRwOi8vd3d3LnczLm9yZy9HcmFwaGljcy9TVkcvMS4xL0RURC9zdmcxMS5kdGQiPjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgdmVyc2lvbj0iMS4xIiBpZD0ibWRpLWFycm93LWJvdHRvbS1yaWdodC1ib2xkLW91dGxpbmUiIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNOS44OCwxNS41NEw0LjIyLDkuODhMOS44OCw0LjIyTDE1LjU0LDkuODhMMTkuNzgsNS42NFYxOS43OEg1LjY0TDkuODgsMTUuNTRNMTcuNjYsMTAuNTlMMTUuNTQsMTIuNzFMOS44OCw3LjA1TDcuMDUsOS44OEwxMi43MSwxNS41NEwxMC41OSwxNy42NkgxNy42NlYxMC41OVoiIC8+PC9zdmc+",
   "name": "PointAndOrbit",
   "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/arrow-bottom-right-bold-outline.svg",
-  "shortDescription": "Point an object toward a position and Orbit around a position.",
-  "version": "1.0.0",
+  "shortDescription": "Point an object toward a position or angle while orbiting that object around a position.",
+  "version": "1.1.0",
+  "description": [
+    "# User can",
```

Comment:
The wiki uses `#` for page title. Titles in description should be `##`, but "User can" will look a bit silly in a summary. I think it's better to not use titles in descriptions.

Also, either use "you can" or "It allows" instead of "user can". You won't see "user" anywhere in the wiki (if you do, please report it).